src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -59,12 +64,75 @@ public Task AddComponentAsync(Type componentType, ParameterView parameters, stri
         return RenderRootComponentAsync(componentId, parameters);
     }
 
+    // This ID needs to be kept in sync with the Blazor JS implementation.
+    protected override int AllocateRendererId() => 1;
```

Comment:
Would it make sense to call this `GetWebRendererId`? Using the term "allocate" suggests distinct values would be returned.

Reply:
Also, perhaps nitpicky, but there could be a shared-source enum giving these IDs so it will be more obvious to us in the future where they are centrally organized and we won't have to worry about how to be sure the values are unique.

Reply:
I agree, `GetWebRendererId` sounds better overall and is more descriptive of what's actually happening.

I think the reason I decided on the "allocate" terminology is that there's already a `RendererId` property that has a getter. I know not many people are going to be actually interacting with the renderer directly, but I thought having both `RendererId` and `Get{Web}RendererId()` could be confusing (i.e., which one should I use if I'm trying to get the renderer ID?). The "allocate" term makes it a little scarier in the list of intellisense options. But that name does seem misleading.

Maybe `AssignRendererId()`? But that's also not accurate.

Agh, I'm overthinking it. `GetWebRendererId()` sound fine to me 🙂